docs/workflows/api-token.md

```diff
@@ -114,7 +114,7 @@ import (
 func main() {
 client := &http.Client{}
 var data = strings.NewReader(`grant_type=client_credentials&scope=SCOPES`)
```

Comment:
```suggestion
data := strings.NewReader(`grant_type=client_credentials&scope=${SCOPES}`)
```

Similar changes for the Golang code snippets elsewhere too.

And should all the variables in all the code snippets be updated to use `${}`? E.g.`${TENANT_ID}`?

Reply:
I might make a separate MR for `${}` because there's a lot of other files that we would need to update too.

Reply:
Got it, are the Golang snippet changes (to use the shorthand variable declaration `:=`) going to be in this MR?

Reply:
Probably change all code snippet stuff in another MR. We're also looking into finding a component that would automatically generate snippets for us from a curl. If you have any ideas, open to hearing them!

Reply:
Sounds good! I don't have any prior experience with any tool like that but I'll keep an eye out.